examples/title-and-descriptin-with-markdown.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: title-and-description-with-markdown-
+  labels:
+    workflows.argoproj.io/archive-strategy: "false"
+  annotations:
+    workflows.argoproj.io/title: "**Test Title**"
```

Comment:
there should be a comment specifying the version this is applicable to similar to other examples

Reply:
Fixed in #12582